src/en/get-involved.md

```diff
@@ -53,39 +59,32 @@ Provide any of the following for each component or pattern:
 
 <hr />
 
-<gcds-heading tag="h2" margin-top="225">Coming soon</gcds-heading>
+<gcds-heading tag="h2" margin-top="225">What else is coming soon</gcds-heading>
 
 ### Upcoming components and templates
 
-<div>
-  <gcds-heading tag="h4" margin-bottom="0">Notice</gcds-heading>
-  <ul class="mb-300">
-    <li>The notice is a short prominent message to draw attention and inform.</li>
-    <li><strong>Expected release:</strong> Fall 2024</li>
-  </ul>
-</div>
+Here’s what we’ll soon be releasing. More on our roadmap.
 
 <div>
-  <gcds-heading tag="h4" margin-bottom="0">Topic Page Template</gcds-heading>
+  <gcds-heading tag="h4" margin-bottom="0">Header V2</gcds-heading>
```

Comment:
Are we able to label these something different then `V2`? I feel like that is an internal designation and won't mean much to people reading our docs site since we don't version components individually.

Reply:
Yeah we could use update instead